test/evergreen.yml

```diff
@@ -1637,18 +1637,20 @@ tasks:
         vars:
           make_command: make all
       - func: "make check all"
-  
+
+  # Use format.sh to run tests in parallel (x4) for just under two hours (the
+  # default Evergreen timeout) on the higher spec build distros. This allows
+  # us to perform multiple test runs while ensuring a long-running config does
+  # not result in an Evergreen test timeout failure.
   - name: linux-directio
     depends_on:
     - name: compile
     commands:
       - func: "fetch artifacts"
       - func: "compile wiredtiger"
-      - func: "format test"
+      - func: "format test script"
         vars:
-          times: 3
-          config: ../../../test/format/CONFIG.stress
-          extra_args: direct_io=1
+          format_test_script_args: -t 110 -j 4 direct_io=1
```

Comment:
I believe the `-j 4` part has been covered by the `format test script` function, specifically, the `${smp_command|}` argument of the `format.sh` call.

Reply:
You can remove the "-j 4" option as it is already taken care of by ${smp_command|}.

Reply:
The `smp_command` will use the number of processors on the machine as the number of threads to run, and that is not the intent here.

Reply:
smp_command(number of processor from /proc/cpuinfo) is anyways getting used in the "format test script" implementation. 

Copied the below command from your patch build. 
**./format.sh -j $(grep -c ^processor /proc/cpuinfo) -t 110 -j 4 direct_io=1 2>&1** 

So, you don't need to specify it again.

Reply:
I understand, but I do not want to set it to the number of processors on the machine. I want it set to a lower number.